pyproject.toml

```diff
@@ -1,2 +1,11 @@
 [build-system]
-requires = ["setuptools>=40.8.0", "wheel", "pytest-runner~=4.0"]
+requires = ["setuptools>=40.8.0", "wheel"]
+
+[tool.ruff]
+line-length=160
```

Comment:
that's high!

Reply:
Aye, it was the current setting. I just kept it